README-developers.md

```diff
@@ -0,0 +1,113 @@
+# `cspyce` DEVELOPERS' GUIDE
+March 2022
+
+Frank Yellin, PDS Ring-Moon Systems Node, SETI Institute volunteer.
+
+This guide is intended for those who have downloaded the 
+[github source code](https://github.com/SETI/pds-cspyce/) and are modifying
+or extending the `cspyce` toolkit.
+You do not need to read this if you are installing an existing `PyPI`
+distribition of the sources.
+
+## SETTING UP
+
+The root of the github tree is called `pds-cspyce`.
+When we use `pds-cspcyce` in this document, we are referring to that directory,
+wherever it happens to be on your computer.
+
+The github source code intentionally does not include the source code to
+cspice itself.
+You should download the cspice sources from XXXX.
```

Comment:
You download CSPICE from: https://naif.jpl.nasa.gov/naif/toolkit.html

You might want to tell them which version to download, if it matters

Reply:
Fixed.